electron/launcher/main-process/electron.js

```diff
@@ -24,7 +27,10 @@ function createWindow() {
         resizable: false,
         frame: false,
         icon: path.join(__dirname, 'dietstory-desktop-icon.ico'),
-        webPreferences: {nodeIntegration: true}
+        webPreferences: {
+            nodeIntegration: true,
+            webSecurity: false
```

Comment:
Are we disabling web security for the purpose of displaying local images?

https://stackoverflow.com/questions/48777336/is-disabling-websecurity-on-the-electron-window-the-only-way-to-display-local-im

Looks like this is a development only issue?